docs/troubleshooting/index.rst

```diff
@@ -0,0 +1,23 @@
+Troubleshooting
+===============
+
+Common problems and how to identify and deal with them can be found here,
+as well as how to file a bug if your issue was not covered.
+
+.. toctree::
```

Comment:
I am not sure whether to show `toctree` on the page or just on the sidebar (aka add `:hidden:` here)